pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2528,6 +2528,27 @@ with haskellLib;
       '';
   }) super.hermes-json;
 
+
+  # hexstring is not compatible with newer versions of base16-bytestring
+  # See https://github.com/solatis/haskell-hexstring/issues/3
+  hexstring = appendPatch
+    (pkgs.fetchpatch {
+        name = "fix-base16-bytestring-compat";
+        url = "https://github.com/solatis/haskell-hexstring/pull/4/commits/4f0a27c64ecb4a767eeea2efebebfd7edba18de0.patch";
+        hash = "sha256-DHT566Ov1D++1VNjUor9xSeOsuSi2LPiIAGT55gqr8s=";
+      })
+    # Source in the original tarball is in CRLF so patch do not apply
+    # This is the same commit as the release 0.11.1 on hackage
+    (overrideSrc {
+      version = "0.11.1";
+      src = pkgs.fetchFromGitHub {
+        owner = "solatis";
+        repo = "haskell-hexstring";
+        rev = "7b078c902a95026b3595fbe4749438993626dd77";
+        hash = "sha256-wCFfsQfJ5cPWHQ3Bt/HIsvAppQKxWyiomqpHBvs6/uc=";
+      };
+  } super.hexstring);
```

Better solution for the CRLF.

```suggestion
    (overrideCabal (drv: {
      prePatch = ''
        ${drv.prePatch or ""}
        ${lib.getExe' pkgs.buildPackages.dos2unix "dos2unix"} *.cabal
     '';
    }) super.hexstring);
```